app/helpers/home_helper.rb

```diff
@@ -0,0 +1,2 @@
+module HomeHelper
+end
```

Comment:
Pls remote it and try to not use helpers at all. Helpers is very hard thing to test.